docs/api/rpc.md

```diff
@@ -1,974 +1,3433 @@
gA0UNASAEIAM2ArgIIARBADsBtAggBEEBNgKwCCAEQZaRwYAANgKsCCAEQQI7AagIQQQhBgwBC0ECIQYgBEECOwGQCAJAIANFDQAgBCADNgKgCCAEQQA7AZwIIARBAjYCmAggBEGUkcGAADYClAgMAQtBASEGIARBATYCmAggBEHo+cCAADYClAgLIAQgDjYCwAgLIARBzAhqIAY2AgAgBCAINgLECCAEIARBkAhqNgLICCAAIARBwAhqELqHgIAAIQYgBEHwCGokgICAgAAgBg8LIAxBgAhB6JHBgAAQk4eAgAAAC0G8jsGAAEEhQdSQwYAAEJGHgIAAAAtB5JDBgABBH0GEkcGAABCRh4CAAAALqgoHAX8BfgF/AX4BfwN+A38jgICAgABBgAFrIgQkgICAgAACQAJAIAG9IgVC////////////AINQRQ0AQQQhBgwBCyAFQv////////8HgyIHQoCAgICAgIAIhCAHQgGGIAVCNIinQf8PcSIIGyIJQgGDIQoCQCAFQoCAgICAgID4/wCDIgtQDQACQCALQoCAgICAgID4/wBSDQBBA0ECIAdQGyEGDAILQoCAgICAgIAgIAlCAYYgCUKAgICAgICACFEiDBshCUICQgEgDBshByAKp0EBcyEGQct3Qcx3IAwbIAhqIQgMAQsgCEHNd2ohCCAKp0EBcyEGQgEhBwsgBEEIakEEaiAEQSBqQQRqLQAAIgw6AAAgBCAEKAAgIg02AgggBEH/AGogDDoAACAEIAg7AXggBCAHNwNwIARCATcDaCAEIAk3A2AgBCANNgB7IAQgBjoAegJAAkACQAJAAkACQAJAAkACQCAGQX5qIgZBAyAGQf8BcSIMQQNJG0H/AXEiBkECSw0AAkAgBg4DAgEAAgtB6PnAgAAhDEEAIQgCQAJAAkACQAJAIAJB/wFxDgQEAAECBAtBl5HBgABB6PnAgAAgBUIAUxshDCAFQj+IpyEIDAMLQZiRwYAAIQwMAQtBl5HBgABBmJHBgAAgBUIAUxshDAtBASEIC0ECIQYgBEECOwEgIANFDQMgBCADNgIwIARBADsBLCAEQQI2AiggBEGUkcGAADYCJAwEC0GXkcGAAEHo+cCAACAFQgBTIggbQZeRwYAAQZiRwYAAIAgbIAJB/wFxIg1BAkkbIQJBASEGIAggDUEBS3IhCCAMQQJNDQEgBEEgaiAEQeAAaiAEQQhqQREQnoeAgAACQAJAIAQoAiBBAUYNACAEIARB4ABqIARBCGpBERCbh4CAACAELwEEIQYgBCgCACEMDAELIARBKGovAQAhBiAEKAIkIQwLIAxBEk8NBSAMRQ0GIAQtAAhBMUkNBwJAAkACQAJAIAZBEHRBEHUiDUEBSA0AQQIhBiAEQQI7ASAgBCAEQQhqNgIkIAwgDU0NASAEQQI7ATggBEEBNgI0IARBlpHBgAA2AjAgBEECOwEsIAQgDTYCKCAEIARBCGogDWo2AjwgBCAMIA1rIg42AkBBAyEGIA4gA08NAyAEQQA7AUQgBCADIAxrIA1qNgJIDAILIARBAjsBOCAEQQI2AiggBEGUkcGAADYCJCAEQQI7ASAgBEEAOwEsIARBACANayIONgIwIAQgDDYCQCAEIARBCGo2AjxBAyEGIAwgA08NAiADIAxrIgwgDk0NAiAEQQA7AUQgBCAMIA1qNgJIDAELIARBADsBLCAEIAw2AiggBCANIAxrNgIwIANFDQEgBCADNgJIIARBADsBRCAEQQE2AkAgBEGWkcGAADYCPCAEQQI7ATgLQQQhBgsgBCACNgJQDAQLIARBAzYCKCAEQa+RwYAANgIkIARBAjsBICAEQej5wIAANgJQQQEhBkEAIQgMAwsgBEEDNgIoIARBrJHBgAA2AiQgBEECOwEgIAQgAjYCUAwCC0EBIQYgBEEBNgIoIARB6PnAgAA2AiQLIAQgDDYCUAsgBEHcAGogBjYCACAEIAg2AlQgBCAEQSBqNgJYIAAgBEHQAGoQuoeAgAAhBiAEQYABaiSAgICAACAGDwsgDEERQZyRwYAAEJOHgIAAAAtBvI7BgABBIUHUkMGAABCRh4CAAAALQeSQwYAAQR9BhJHBgAAQkYeAgAAAC5UCAQJ/I4CAgIAAQRBrIgIkgICAgAAgAkEANgIMAkACQAJAAkAgAUGAAUkNACABQYAQSQ0BIAJBDGohAyABQYCABE8NAiACIAFBP3FBgAFyOgAOIAIgAUEMdkHgAXI6AAwgAiABQQZ2QT9xQYABcjoADUEDIQEMAwsgAiABOgAMIAJBDGohA0EBIQEMAgsgAiABQT9xQYABcjoADSACIAFBBnZBwAFyOgAMIAJBDGohA0ECIQEMAQsgAiABQT9xQYABcjoADyACIAFBEnZB8AFyOgAMIAIgAUEGdkE/cUGAAXI6AA4gAiABQQx2QT9xQYABcjoADUEEIQELIAAgAyABELKHgIAAIQEgAkEQaiSAgICAACABC3EBAX8jgICAgABBIGsiAiSAgICAACACIAA2AgQgAkEIakEQaiABQRBqKQIANwMAIAJBCGpBCGogAUEIaikCADcDACACIAEpAgA3AwggAkEEakGolsGAACACQQhqEKGHgIAAIQEgAkEgaiSAgICAACABCxEAIAAoAgAgASACELKHgIAAC5wCAQJ/I4CAgIAAQRBrIgIkgICAgAAgACgCACEAIAJBADYCDAJAAkACQAJAIAFBgAFJDQAgAUGAEEkNASACQQxqIQMgAUGAgARPDQIgAiABQT9xQYABcjoADiACIAFBDHZB4AFyOgAMIAIgAUEGdkE/cUGAAXI6AA1BAyEBDAMLIAIgAToADCACQQxqIQNBASEBDAILIAIgAUE/cUGAAXI6AA0gAiABQQZ2QcABcjoADCACQQxqIQNBAiEBDAELIAIgAUE/cUGAAXI6AA8gAiABQRJ2QfABcjoADCACIAFBBnZBP3FBgAFyOgAOIAIgAUE
```

Comment:
probably doesn't make sense to include everything here. It is too large

Reply:
Yeah, I was wondering about this one. I'll shorten it up :)